common/s2n-codec/src/decoder/checked_range.rs

```diff
@@ -76,3 +77,25 @@ impl fmt::Debug for CheckedRange {
         write!(f, "{}..{}", self.start, self.end)
     }
 }
+/// The following harnesses were automatically generated by an
+/// experimental tool developed by the Kani team.
```

Comment:
Is the plan to merge the PR with these comments? Not sure they're super helpful? Also there should be a space above separating the last item.

Reply:
I've removed these stock comments.